Tuple members must all have names or all not have names

According to TS docs it's not allowed to use a variation of labeled and unlabeled tuples. However this is the case for the type declaration of the function [`calc()` in index.d.ts](https://github.com/NickLJudy/numeral-calc/blob/main/src/index.d.ts#L15-L20)

Please consider to either name all the tuple elements or none. The current implementation causes a compile error.

```
numeral-calc/src/index.d.ts:18:5 - error TS5084: Tuple members must all have names or all not have names.

18     '+' | 'and' | 'plus' | 'add' | '-' | 'minus' | 'sub' | 'subtract' | '*' | 'x' | 'multi' | 'multiply' | '/' | 'divide'
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Reference to the documentation:**

 - [Labeled Tuple Elements](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements)

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -15,7 +15,7 @@ export declare function divide(
 export declare function calc(
   ...paramArr: [
     ...numArr: string[] | number[] | bigint[],
-    '+' | 'and' | 'plus' | 'add' | '-' | 'minus' | 'sub' | 'subtract' | '*' | 'x' | 'multi' | 'multiply' | '/' | 'divide'
+    operator: '+' | 'and' | 'plus' | 'add' | '-' | 'minus' | 'sub' | 'subtract' | '*' | 'x' | 'multi' | 'multiply' | '/' | 'divide'
   ]
 ): string;
 

